hondana/collections.py

```diff
@@ -48,6 +48,7 @@
     from .user import User
 
 __all__ = (
+    "BaseCollection",
```

Comment:
Do we need access to this from the main package? `hondana.BaseCollection` would now be exposed. Let me know your thoughts!

Reply:
Sphinx fails if I don't do this. I'm not aware enough about Sphinx to fix this.

Reply:
No worries, I can resolve this later.